### PR TITLE
Retry on 500 server error raised by gql

### PR DESCRIPTION
**Pull request by norrishd**
The algo has somewhat randomly started getting a lot of:
`Exception: {'statusCode': 500, 'errorCode': 'SERVER_ERROR', 'message': 'Internal server error'}`

These are raised by the `gql` library that `seer-py` uses internally and IDK, maybe they originate from our GQL server? Or further downstream as rate limits from S3? Opinions vary and I'm unclear. See also https://github.com/seermedical/seer-algo/issues/113.

In any case, pretty sure `seer-py` isn't catching these or retrying, so this is a tiny change to add this as an error to retry for. In the best case maybe this will largely fix things, and at the worst, will probably just drag things out for a few minutes before an instance turns into a zombie. Probably a worthwhile first step before taking a more drastic measure like switching to S3 direct?

P.s. my `pre-commit` YAPFed a bunch of bad indentation that had crept into the file, sorry that obscures the actual meaningful change.

### CODE REVIEW

File: seerpy/seerpy.py [hunk 1]
```diff
@@ -217,7 +221,6 @@ def get_paginated_response(self, query_string, variable_values, limit, object_pa
         offset = variable_values.get('offset', 0)  # Try get offset from variable values if exists
         result = []
         total_items_returned = 0
-        remaining_items = 0
```
Comment (norrishd) [inline seerpy/seerpy.py:220]: This is always redefined in the loop before being used, so isn't needed

**Comment by ronrest** [comment]
Looks good to me

**Review by ronrest — APPROVED** [review]
looks good to me

**Review by matias-seer — APPROVED** [review]
looks good

**Pull request merged**